runners/flink/src/main/java/org/apache/beam/runners/flink/translation/wrappers/streaming/DoFnOperator.java

```diff
@@ -285,7 +260,7 @@ public void open() throws Exception {
         mainOutputTag,
         additionalOutputTags,
         stepContext,
-        aggregatorFactory,
+        null,
```

Comment:
I literally copied this from the PR to remove Aggregators of ApexRunner. I suppose this is a pending step of the removal on runners-core.